src/Commands/DrupalSettingsCommands.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
This file should be deleted in favour of src/Commands/DrupalSettingsCommands.php?